lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -522,12 +529,11 @@ void SwiftLanguageRuntimeImpl::SetupReflection() {
 
```

Comment:
m_modules_to_add has been cleared when you reach the Progress initializer. I think you want the snapshot here?